doc/theory/spherical.rst

```diff
@@ -26,31 +26,30 @@ OP
 radial distance
    or radius.  The Euclidean length of OP.
 
-z axis
-   The vertical of the sphere.  If we consider the sphere as a globe,
-   then the z axis runs from south to north.  This is the zenith direction of the sphere. 
+z-axis
+   The vertical of the sphere.  If we consider the sphere like a globe,
```

Comment:
I actually prefer this one as it already is.

```suggestion
   The vertical of the sphere.  If we consider the sphere as a globe,
```

Reply:
Oh sure. I will do the changes.